packages/parent-bridge/contracts/audit-delay/src/entry.rs

```diff
@@ -0,0 +1,59 @@
+// Import from `core` instead of from `std` since we are in no-std mode
+use core::result::Result;
+use core::convert::TryInto;
+
+// Import heap related library from `alloc`
+// https://doc.rust-lang.org/alloc/index.html
+use alloc::{vec, vec::Vec};
+
+// Import CKB syscalls and structures
+// https://nervosnetwork.github.io/ckb-std/riscv64imac-unknown-none-elf/doc/ckb_std/index.html
+use ckb_std::{
+    ckb_constants::Source,
+    debug,
+    high_level::{load_script, load_tx_hash, load_header, load_cell_lock_hash},
+    ckb_types::{bytes::Bytes, prelude::*},
+};
+
+use crate::error::Error;
+
+pub fn main() -> Result<(), Error> {
+    let input0_lock_hash = load_cell_lock_hash(0, Source::Input)?;
+    let script = load_script()?;
+    let args: Bytes = script.args().unpack();
+
+    if args.len() != 72 {
+        return Err(Error::WrongScriptArgsLength);
+    }
+
+    let trustee_lock_hash = args.slice(0..32);
+    let owner_lock_hash = args.slice(32..64);
+
+    if Bytes::from(&input0_lock_hash[..]) == trustee_lock_hash {
+        return Ok(());
+    }
+
+    let input_header = load_header(0, Source::GroupInput)?;
+    let input_header_timestamp = input_header.raw().timestamp().unpack();
+    debug!("{:?}", input_header_timestamp);
```

Comment:
Can we remove all debug statements or have them as comments?

Reply:
Yeah, I forgot to remove them myself.